sockeye/layers_pt.py

```diff
@@ -101,17 +102,16 @@ def __init__(self,
             self.weight = weight
         self.bias = pt.nn.Parameter(pt.Tensor(vocab_size))
 
-        self.previous_slice_ids = None  # type: Optional[pt.Tensor]
-        self.reduced_weight_bias = None  # type: Optional[Tuple[pt.Tensor, pt.Tensor]]
+        self.previous_slice_ids: Optional[pt.Tensor] = pt.empty(0)
+        self.reduced_weight: Optional[pt.Tensor] = pt.empty(0)
+        self.reduced_bias: Optional[pt.Tensor] = pt.empty(0)
```

Comment:
Do these still need to be `Optional` (or have type annotations at all) now that they're created as `Tensor`s?

Reply:
Good catch, we can remove the type annotation.